src/routes/sitemaps.ts

```diff
@@ -0,0 +1,28 @@
+import { FastifyInstance } from 'fastify';
+import { Post, PostType } from '../entity';
+import createOrGetConnection from '../db';
+
+export default async function (fastify: FastifyInstance): Promise<void> {
+  fastify.get('/posts.txt', async (req, res) => {
+    const con = await createOrGetConnection();
+    const query = con
+      .createQueryBuilder()
+      .select(`concat('${process.env.COMMENTS_PREFIX}', '/posts/', id)`, 'url')
+      .from(Post, 'p')
+      .where('type NOT IN (:...types)', { types: [PostType.Welcome] })
+      .andWhere('NOT private')
+      .andWhere('NOT banned')
+      .andWhere('NOT deleted')
+      .andWhere('"createdAt" > current_timestamp - interval \'90 day\'')
+      .orderBy('"createdAt"', 'DESC')
+      .limit(50_000);
```

Comment:
We might have to consider paginated sitemaps for the amount we have.

Think most parsers struggle with big amounts in root, making the expectef behaviour not accomplished.

Should be a testing tool for this

Reply:
Yes, I checked 50k is the max